pkgs/sagemath-doc-html/meson.build

```diff
@@ -0,0 +1,21 @@
+project('sagemath-doc-html',
+        version: run_command('cat',
+                             join_paths(meson.current_source_dir(), 'VERSION.txt'),
+                             check: true).stdout().strip()
+)
+
+docdir = get_option('datadir') / 'share' / 'doc' / 'sage'
+
+html = custom_target('HTML documentation',
```

Comment:
`run_command` is definitely not the right choice because it runs at "setup" time, not at "build" time.

Reply:
So you want to run this during the build time? I got confused by the install command below. In this case, custom target is indeed the correct command, but you need to specify the correct output folder name - otherwise the output is not installed correctly. Then you can remove the manual install of the output as well.

Reply:
Nonsense. We've tested that the wheel is built correctly.

Reply:
What is nonsense? You can test it yourself, it does not work without the install_subdir command (which is not required if you would just specify the correct output).

Reply:
> it does not work without the install_subdir command

Yes, if you remove something from the working solution, it no longer works.